packages/react-core/src/layouts/Flex/Flex.tsx

```diff
@@ -3,25 +3,368 @@ import { css } from '@patternfly/react-styles';
+      | 'spacerXs'
+      | 'spacerSm'
+      | 'spacerMd'
+      | 'spacerLg'
+      | 'spacerXl'
+      | 'spacer2xl'
+      | 'spacer3xl'
+      | 'spacer4xl';
+    md?:
+      | 'spacerNone'
+      | 'spacerXs'
+      | 'spacerSm'
+      | 'spacerMd'
+      | 'spacerLg'
+      | 'spacerXl'
+      | 'spacer2xl'
+      | 'spacer3xl'
+      | 'spacer4xl';
+    lg?:
+      | 'spacerNone'
+      | 'spacerXs'
+      | 'spacerSm'
+      | 'spacerMd'
+      | 'spacerLg'
+      | 'spacerXl'
+      | 'spacer2xl'
+      | 'spacer3xl'
+      | 'spacer4xl';
+    xl?:
+      | 'spacerNone'
+      | 'spacerXs'
+      | 'spacerSm'
+      | 'spacerMd'
+      | 'spacerLg'
+      | 'spacerXl'
+      | 'spacer2xl'
+      | 'spacer3xl'
+      | 'spacer4xl';
+    '2xl'?:
+      | 'spacerNone'
+      | 'spacerXs'
+      | 'spacerSm'
+      | 'spacerMd'
+      | 'spacerLg'
+      | 'spacerXl'
+      | 'spacer2xl'
+      | 'spacer3xl'
+      | 'spacer4xl';
+  };
+  /** Space items at various breakpoints */
+  spaceItems?: {
+    default?:
+      | 'spaceItemsNone'
+      | 'spaceItemsXs'
+      | 'spaceItemsSm'
+      | 'spaceItemsMd'
+      | 'spaceItemsLg'
+      | 'spaceItemsXl'
+      | 'spaceItems2xl'
+      | 'spaceItems3xl'
+      | 'spaceItems4xl';
+    sm?:
+      | 'spaceItemsNone'
+      | 'spaceItemsXs'
+      | 'spaceItemsSm'
+      | 'spaceItemsMd'
+      | 'spaceItemsLg'
+      | 'spaceItemsXl'
+      | 'spaceItems2xl'
+      | 'spaceItems3xl'
+      | 'spaceItems4xl';
+    md?:
+      | 'spaceItemsNone'
+      | 'spaceItemsXs'
+      | 'spaceItemsSm'
+      | 'spaceItemsMd'
+      | 'spaceItemsLg'
+      | 'spaceItemsXl'
+      | 'spaceItems2xl'
+      | 'spaceItems3xl'
+      | 'spaceItems4xl';
+    lg?:
+      | 'spaceItemsNone'
+      | 'spaceItemsXs'
+      | 'spaceItemsSm'
+      | 'spaceItemsMd'
+      | 'spaceItemsLg'
+      | 'spaceItemsXl'
+      | 'spaceItems2xl'
+      | 'spaceItems3xl'
+      | 'spaceItems4xl';
+    xl?:
+      | 'spaceItemsNone'
+      | 'spaceItemsXs'
+      | 'spaceItemsSm'
+      | 'spaceItemsMd'
+      | 'spaceItemsLg'
+      | 'spaceItemsXl'
+      | 'spaceItems2xl'
+      | 'spaceItems3xl'
+      | 'spaceItems4xl';
+    '2xl'?:
+      | 'spaceItemsNone'
+      | 'spaceItemsXs'
+      | 'spaceItemsSm'
+      | 'spaceItemsMd'
+      | 'spaceItemsLg'
+      | 'spaceItemsXl'
+      | 'spaceItems2xl'
+      | 'spaceItems3xl'
+      | 'spaceItems4xl';
+  };
+  /** Whether to add flex: grow at various breakpoints */
+  grow?: {
+    default?: 'grow';
+    sm?: 'grow';
+    md?: 'grow';
+    lg?: 'grow';
+    xl?: 'grow';
+    '2xl'?: 'grow';
+  };
+  /** Whether to add flex: shrink at various breakpoints */
+  shrink?: {
+    default?: 'shrink';
+    sm?: 'shrink';
+    md?: 'shrink';
+    lg?: 'shrink';
+    xl?: 'shrink';
+    '2xl'?: 'shrink';
+  };
+  /** Value to add for flex property at various breakpoints */
+  flex?: {
+    default?: 'flexDefault' | 'flexNone' | 'flex_1' | 'flex_2' | 'flex_3' | 'flex_4';
```

Comment:
There are also these flex_1 etc values and the width_33 values above, WDYT, camelCase them as well? These and the width values do look very readable, so either way is fine with me

Reply:
Ah, I forgot about these guys! I think they're fine with underscores. The `width33` in particular is harder to read.